app.js

```diff
@@ -1,3 +1,4 @@
+require('babel-core/register');
```

Comment:
Only used for dev only. Need to find a way to create server side component later.